CMakeLists.txt

```
@@ -1,26 +1,38 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.19)
 project(poplar_trie)
 
-if (NOT CMAKE_BUILD_TYPE)
+option(POPLAR_FORCE_RELEASE "Force a release build" OFF)
+option(POPLAR_DISABLE_SSE4_2 "Disable SSE 4.2" OFF)
+option(POPLAR_VERBOSE "Emit diagnostic info" OFF)
```

```suggestion
option(POPLAR_VERBOSE "Emit diagnostic info" ON)
```

The primary objective of this library is to benchmark data structures in experimental studies.
So, I hope users will pay attention to the compilation flags.